### PR TITLE
add new Schema

### PR DESCRIPTION
**Pull request by eidan66**

### CODE REVIEW

File: server/models/Item.js [hunk 1]
```diff
@@ -1,14 +1,9 @@
 const mongoose = require('mongoose')
 const ItemSchema = new mongoose.Schema({
-    itemID: {
-        type: String,
-        default: Schema.Types.ObjectId,
-    },
     itemName: {
```
Comment (yonatansnir) [inline server/models/Item.js]: You're already on the Item collection. You don't really need to write "itemName". We know it's the name of the item.

don't you think so?

File: server/models/User.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+const mongoose = require('mongoose')
+const UserSchema = new mongoose.Schema({
+    firstName: {
```
Comment (yonatansnir) [inline server/models/User.js]: I think It's better to write just "full_name" instead of separated. It's easier to filter on just one field.

**Review by yonatansnir — COMMENTED** [review]
Why don't you use the style of SQL with separate word by _ ?

File: server/models/Item.js (hunk 1 above)
Comment (eidan66) [reply server/models/Item.js]: I'll fix it.

**Review by yonatansnir — COMMENTED** [review]
good job

**Pull request merged**